### PR TITLE
Adds links to Logstash plugins under the Integrations page.

### PR DESCRIPTION
**Pull request by palecur**

### CODE REVIEW

**Comment by markwalkom** [comment]
LGTM :D

File: docs/plugins/integrations.asciidoc [hunk 1]
```diff
@@ -29,13 +28,24 @@ make it easier to work with Elasticsearch.
...
 [[data-integrations]]
 === Data import/export and validation
 
-NOTE: Rivers were used to import data from external systems into
-Elasticsearch, but they are no longer supported in Elasticsearch 2.0.
+NOTE: Rivers were used to import data from external systems into Elasticsearch prior to the 2.0 release. Elasticsearch 
+releases 2.0 and later do not support rivers.
+
+[float]
+==== Supported by the community:
```
Comment (dadoonet) [inline docs/plugins/integrations.asciidoc]: We should use another Title here.
Also they are not really plugins you can add to elasticsearch but plugins which connect to elasticsearch.

May be with a title like Logstash: river replacement...

WDYT?

Comment (palecur) [inline docs/plugins/integrations.asciidoc]: Is it really a river replacement, though? Rivers are their own functionality. While these aren't ES plugins, they are integrations between ES and LS, so I think they fall under the general title.

Comment (dadoonet) [inline docs/plugins/integrations.asciidoc]: @palecur Ignore me! I'm sorry. I totally misread this section...

**Comment by dadoonet** [comment]
LGTM

File: docs/plugins/integrations.asciidoc (hunk 1 above)
Comment (palecur) [inline docs/plugins/integrations.asciidoc]: Aw, that's fine! I'll go ahead and merge this.

**Pull request merged**